src/CommunityToolkit.Maui/Behaviors/Validators/ValidationBehavior.shared.cs

```diff
@@ -397,7 +397,7 @@ async ValueTask UpdateStateAsync(VisualElement? view, ValidationFlags flags, boo
 				IsValid = false;
 				IsRunning = false;
 
-				Trace.WriteLine(ex);
+				Trace.TraceError(ex.StackTrace);
```

Comment:
I think we should rely on the full information

```suggestion
				Trace.TraceError(ex);
```